<ui version="4.0" >
 <class>NetworkSettingsDlg</class>
 <widget class="QWidget" name="NetworkSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Network</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../simon.qrc" >:/images/icons/network-receive.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string>Allgemein</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Timeout (Millisekunden):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbTimeout" >
        <property name="minimum" >
         <number>500</number>
        </property>
        <property name="maximum" >
         <number>60000</number>
        </property>
        <property name="value" >
         <number>5000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbJulius" >
     <property name="title" >
      <string>Juliusd</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTableWidget" name="twJuliusAddresses" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text" >
          <string>Adresse</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Port</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QToolButton" name="tbMoveUp" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../simon.qrc" >:/images/icons/go-up.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbMoveDown" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../simon.qrc" >:/images/icons/go-down.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbAddAddress" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Hinzuf√ºgen</string>
          </property>
          <property name="icon" >
           <iconset resource="../simon.qrc" >:/images/icons/list-add.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbDeleteAddress" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Entfernen</string>
          </property>
          <property name="icon" >
           <iconset resource="../simon.qrc" >:/images/icons/list-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../simon.qrc" />
 </resources>
 <connections/>
</ui>
